src/Bridges/Constraint/Constraint.jl

```diff
@@ -7,6 +7,8 @@
 module Constraint
 
 import LinearAlgebra
+import MutableArithmetics
+const MA = MutableArithmetics
```

Comment:
I'm going to enforce my rule of no new acronyms other than MOI. They just make the code harder to read. You should be able to read each bridge file in isolation without needing to go looking for what each thing is. I know that _we_ know what MA is, but it just makes things harder for others coming along later.